### ISSUE TITLE
value for Dropdown question with choicesByUrl isn't returned in survey.data

### ISSUE THREAD

**Issue by nelero** [andrewtelnov/surveyjs#138]
When you use dropdown question with choicesByUrl, survey.data doesn't contain the value selected.
(tested with knockout integration)

**Comment by andrewtelnov** [comment]
@nelero Could you show your survey json? 
It works fine in the example: http://surveyjs.org/examples/questiontype-dropdownrestfull.html

Thank you,
Andrew

**Comment by nelero** [comment]
Ok, you're right 👍 selected value is ok.
But when i want to select a value in this dropdown, via survey.data = {"Country":"FR"};
i always stay on choose.
```
Survey.Survey.cssType = "bootstrap";
var surveyJson = {
 pages: [
  {
   name: "about_you",
   questions: [
    {
     type: "dropdown",
     choicesByUrl: {
      url: "https://api.myjson.com/bins/1e5ch",
      path: "RestResponse;result",
      valueName: "alpha_2_code",
      titleName: "name"
     },
     isRequired: true,
     name: "Country",
     startWithNewLine: false,
     title: "Country",
     
    }
 ]
};
var survey = new Survey.Survey(surveyJson);
survey.data = {"Country":"FR"};
survey.showProgressBar = "top";
 
 survey.render("mySurveyJSName");
```

**Comment by andrewtelnov** [comment]
@nelero Indeed the dropdown is empty and knockout set value to underfined, question get that notification and reset the value. After loading the data, the value is empty.
I have fixed the issue.
Thank you,
Andrew

**Comment by nelero** [comment]
@andrewtelnov : thanks for the fix. How can i get the latest survey.knockout.js to test it ?
thanks

**Comment by andrewtelnov** [comment]
@nelero I have just published a new version, 0.10.2, you may use it.

Thank you,
Andrew
